src/SFA.DAS.AANHub.Application/Partners/CreatePartnerMemberCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using MediatR;
+using SFA.DAS.AANHub.Application.Common.Commands;
+using SFA.DAS.AANHub.Application.Common.Validators.RequestedByMemberId;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+using SFA.DAS.AANHub.Domain.Entities;
+using static SFA.DAS.AANHub.Domain.Common.Constants;
+
+namespace SFA.DAS.AANHub.Application.Partners
+{
+    public class CreatePartnerMemberCommand : CreateMemberCommandBase, IRequest<ValidatableResponse<CreatePartnerMemberCommandResponse>>, IRequestedByMemberId
+    {
+        public string UserName { get; set; } = null!;
+        public string Organisation { get; set; } = null!;
+        public Guid? RequestedByMemberId { get; set; }
+
+        public static implicit operator Member(CreatePartnerMemberCommand command) => new()
+        {
+            Id = command.Id,
+            UserType = MembershipUserType.Admin,
```

Comment:
Is the Admin user type correct?

Reply:
No, well spotted. This has been updated to `.Partner`